packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -76,15 +77,33 @@ export class EpochTracker implements IPersistedFileCache {
     ): Promise<any> {
         try {
             const value: IVersionedValueWithEpoch = await this.cache.get(this.fileEntryFromEntry(entry));
+            // Version check
             if (value === undefined || value.version !== persistedCacheValueVersion) {
                 return undefined;
             }
+            // Epoch check
             assert(value.fluidEpoch !== undefined, 0x1dc /* "all entries have to have epoch" */);
             if (this._fluidEpoch === undefined) {
                 this.setEpoch(value.fluidEpoch, true, "cache");
             } else if (this._fluidEpoch !== value.fluidEpoch) {
                 return undefined;
             }
+            // Cache expiry check
+            // Only expire the snapshot cache as the ops cache returns ops for a specific range, meaning old ops
```

Comment:
Does the op cache get also come via here?
This comment is not very clear. I think I understand what you are trying to say but this may not be the right place. I don't see any ranges for ops in the get call here unless I am missing something.